chpl/chpl-service/src/main/java/gov/healthit/chpl/manager/PrecacheableDimensionalDataManager.java

```diff
@@ -193,12 +193,13 @@ public PrecacheableDimensionalDataManager(
     }
 
     @Transactional
-    @Cacheable(CacheNames.DEVELOPER_NAMES)
+    @Deprecated
```

Comment:
If we don't need the cache here I would like to see this actually removed and other code changed to use the getDevelopers() method instead

Reply:
I removed this method